### ISSUE TITLE
OEIS

### ISSUE THREAD

**Issue by vinctux** [ipfs/archives#16]
https://oeis.org/

- [x] Download [`stripped.gz`](http://oeis.org/stripped.gz) (20MB) and [`names.gz`](http://oeis.org/names.gz) (5MB)
- [x] Extract, merge and process data
- [x] Publish merged version via [IPFS](gateway.ipfs.io/ipfs/QmSTWTYLiAjWSn4uJbAjDadXeFLYrfeKpShFADNjAUgPdt) (26 MB)

OEIS data under: CC-BY-NC-3.0

**Note:** The OEIS base is updating daily - need for #5

**Comment by jbenet** [comment]
indeed. what hash? can we replicate it?

**Comment by whyrusleeping** [comment]
he hid the hash in an inline link, very tricky: https://gateway.ipfs.io/ipfs/QmSTWTYLiAjWSn4uJbAjDadXeFLYrfeKpShFADNjAUgPdt

**Comment by vinctux** [comment]
Sorry, next time it will be like this: `QmSTWTYLiAjWSn4uJbAjDadXeFLYrfeKpShFADNjAUgPdt`

**Comment by jbenet** [comment]
vinctux: could we replicate it un-archived? so we can browse it on the web and link to internals?

**Comment by jbenet** [comment]
(if permissions and modes matter, dont worry)

**Comment by vinctux** [comment]
@jbenet What exactly are we're expected to replicate? The source `GZip`s are directly accessible on the OEIS server  - and all modification was made by a simple Python script. I think I don't understand completely..

**Comment by davidar** [comment]
:+1: 

@vinctux I think @jbenet means making each oeis entry available as a separate file

**Comment by jbenet** [comment]
yeah, i mean unzipping firxt. though we may just add native zip support. cc @whyrusleeping

**Comment by whyrusleeping** [comment]
i could probably add zip support in a very similar manner to the way i added tar.

**Comment by davidar** [comment]
@whyrusleeping does ipfs have tar support now? Where? 😀
